tests/test_cli.py

```
@@ -288,3 +360,36 @@ def test_cli_no_args(mock_ensure, mock_menu):
+    result = runner.invoke(app, ["list-asset-address-map"])
+
+    assert result.exit_code == 0
+    assert twin in result.stdout
+    assert mainnet in result.stdout
+    assert "Total: 1 twin assets" in result.stdout
+    mock_build_map.assert_called_once()
+
+
+@patch("orion_finance_sdk_py.cli._request_deposit_logic")
+@patch("orion_finance_sdk_py.cli.ensure_env_file")
+def test_cli_request_deposit(mock_ensure, mock_logic):
```

Ruff (0.16.1)

[warning] Unused function argument: `mock_ensure`

(ARG001)